packages/botonic-plugin-contentful/src/render/botonic-converter.ts

```diff
@@ -133,6 +140,17 @@ export class BotonicMsgConverter {
     return this.appendFollowUp(msg, img)
   }
 
+  document(img: cms.Document, delayS = 0): BotonicMsgs {
+    const msg: BotonicMsg = {
+      type: 'document',
+      delay: delayS,
+      data: {
+        document: img.docUrl,
+      },
+    }
+    return this.appendFollowUp(msg, img)
+  }
```

Comment:
img variable should be changed to `doc` or `document`.

Reply:
done thanks!